### ISSUE TITLE
“PlayOnMac.app” can’t be opened because the identity of the developer cannot be confirmed.

### ISSUE THREAD

**Issue by KizzyCode** [PhoenicisOrg/phoenicis#1937]
### Expected behavior
The app opens.

### Actual behavior
The app fails to open with `“PlayOnMac.app” can’t be opened because the identity of the developer cannot be confirmed.`

### Steps to reproduce
Install PlayOnMac either via homebrew-cask or directly from this [DMG](http://repository.playonmac.com/PlayOnMac/PlayOnMac_4.3.4.dmg)

### System information
* PlayOnMac 4.3.4
* macOS Mojave 10.14.4 (18E226)

**Comment by plata** [comment]
@qparis can you check?

**Comment by qparis** [comment]
We've heard about this problem but we have no clue about how reproduce it. It does not happen everytime

**Comment by KizzyCode** [comment]
Have you tried a fresh install of Mojave? It happened on two of my production systems and on a fresh install of Mojave.

**Comment by qparis** [comment]
Hmm yes. Also, the codesign tool seems to reveal that the signature of PlayOnMac.app is correct.
I use the following command:

`codesign -vvvv -s "Developer ID" /Volumes/PlayOnMac/PlayOnMac.app/ -f --deep`

**Comment by KizzyCode** [comment]
Yeah, the signature itself is intact (you also get an error if you damage the bundle).
Maybe the problem is, that you’ve used the “wrong” signing certificate? There are development and distribution certificates (the former is not trusted by different installations). But that’s just a wild guess...

**Comment by qparis** [comment]
Seems that you are right. I think I need to use 3rd Party Mac Developer Application

**Comment by RadhaMadhavaDas** [comment]
I'm having the same issue. I ran POM, it said it had an update available. I downloaded and installed 4.3.4. I'm on OSX 10.14.6. It gives the same error "“PlayOnMac.app” can’t be opened because the identity of the developer cannot be confirmed. - the only option is to hit OK. Any news on adding the new certificate to an incremental release?

**Comment by qparis** [comment]
I will probably do it by the end of the week.

Regards,

**Comment by RadhaMadhavaDas** [comment]
Thank you so much. I appreciate your efforts.

Do you have a patreon page?

**Comment by plata** [comment]
There is https://www.playonlinux.com/en/donate.html. However the site doesn't seem to work currently...

**Comment by qparis** [comment]
It seems that Developer ID is the right certificate after all. I don't understand how I could know

**Comment by n-lavrenko** [comment]
still reproduced and not fixed
High Sierra 10.13.6 (17G65)

**Comment by aldiyarablyazov** [comment]
Reproduced today in macOS Mojave Version 10.14.6. I am the sole user and admin as well as having downloads from anywhere enabled. Downloaded from PlayOnMac website and installed via DMG file.

**Comment by sattellite** [comment]
Today upgraded from Mojave to Catalina 10.15. PlayOnMac worked good on Mojave, reinstalled it from new DMG file on Catalina and I have the same problem.

`sudo spctl --master-disable` not solved problem

**Comment by qparis** [comment]
I think POM 4 won't work with Catalina yet

**Comment by PatMullen** [comment]
Just a note that I am unable to get POM 4.3.4 working on macOS 10.14.6, running on a 2013 21" iMac. Erased the internal drive and installed Mojave so the config is pretty stock. Disabling both Gatekeeper and SIP had no effect. 
 We need POM or equivalent to run WeavePoint, loom software by AVL Looms. AVL said POM would allow WeavePoint to run on a Mac so we went for it. Doesn't look good...

**Comment by qparis** [comment]
I think it should just work on 10.14.6.

**Comment by PatMullen** [comment]
> I think it should just work on 10.14.6.

Glad to know your thoughts, but I'm afraid it doesn't, at least not for me. Two erase-and-install cycles, same result. 😟

**Comment by qparis** [comment]
Can you try to run it from a terminal?

**Comment by PatMullen** [comment]
There are a few ways to launch from the Terminal. So I tried two of them. The first command:
`$ open /Applications/PlayOnMac.app`
resulted in the same error. The second try:
`$ /Applications/PlayOnMac.app/Contents/MacOS/playonmac`
succeeded, sort of. The Terminal had a lot of activity, lots of errors. But in the end, after quitting POM and trying again, Wine did run. WeavePoint appears to have run. It wasn't easy, even for me, the I.T. guy. Unfortunately, I can't expect non-technical end users do deal with launching stuff from the Terminal and dealing with a very un-Mac-like UI. AVL Looms should just admit it's "Windows only" instead of selling their software as a Mac-capable solution. Sorry to vent...

**Comment by qparis** [comment]
> That essentially worked, and it definitely showed me that this can't be given to end users. Sorry to vent...

It definitely can, but may require some extra work in such case. They can contact me if they want a more professional and direct solution for the end user

At the moment, our main focus is to bring back POM on 64bits only OSes. This signature problem will be easy to solve out, no worries.

**Comment by abildintemirlan** [comment]
Had the same issue, try to change your settings back to 'AppStore and identified developers', then try to 'Open Anyway'. After that, just change the settings back to 'Open Anyway'. It should work.

**Comment by jwoertink** [comment]
I just downloaded this for the first time on Catalina. I saw the security message and said allow anyway. Now when I click on the app to open, nothing happens. When I try to run from terminal, I get

```
$ /Applications/PlayOnMac.app/Contents/MacOS/playonmac
env: ./playonmac.real: Bad CPU type in executable
```

I installed from the link on the homepage. Running macOS 10.15.4

**Comment by subzerov** [comment]
> I just downloaded this for the first time on Catalina. I saw the security message and said allow anyway. Now when I click on the app to open, nothing happens. When I try to run from terminal, I get
> 
> ```
> $ /Applications/PlayOnMac.app/Contents/MacOS/playonmac
> env: ./playonmac.real: Bad CPU type in executable
> ```
> 
> I installed from the link on the homepage. Running macOS 10.15.4

Hello @jwoertink , I'm having exactly the same issue. Were you able to solve it?

Thanks!

**Comment by jwoertink** [comment]
Nope. I never got it to work.

**Comment by Yelloboy** [comment]
> Had the same issue, try to change your settings back to 'AppStore and identified developers', then try to 'Open Anyway'. After that, just change the settings back to 'Open Anyway'. It should work.

This way solved my problem! Thx bro:)

**Comment by Aldekein** [comment]
For me it was solved on Catalina 10.15.6 by running:

`xattr -rd com.apple.quarantine /Applications/PlayOnMac.app`

This removed the quarantine flag, and then I was able to run 

`open /Applications/PlayOnMac.app`

Additionally, there's such a button, which I didn't try:

<img width="624" alt="image" src="https://user-images.githubusercontent.com/671454/93590151-b12b7200-f9b6-11ea-9dd4-3a8dded30b42.png">
